features/steps/component.go

```diff
@@ -78,9 +80,13 @@ func (c *Component) initService(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to get config: %w", err)
 	}
+
 	cfg.EncryptionDisabled = true
```

Comment:
Why aren't these config values being passed as environment variables in the component yml file?

Reply:
makes sense, I originally thought we might want to change values depending on the test, but it wasn't the case. I will change it.